### PR TITLE
Defaults group suffix

### PR DESCRIPTION
**Pull request by j-bennet**
This addresses issue #95.

### CODE REVIEW

File: mycli/main.py [hunk 1]
```diff
@@ -166,11 +170,16 @@ def read_my_cnf_files(self, files, keys):
...
+
         def get(key):
-            try:
-                return cnf['client'][key]
-            except KeyError:
-                return None
+            result = None
+            for sect in sections:
+                if sect in cnf and key in cnf[sect]:
+                    result = cnf[sect][key]
```
Comment (amjith) [inline mycli/main.py]: Should this have an early return inside the for loop so it returns as soon as the key is found?

**Comment by amjith** [comment]
This works as expected. I'm going to merge it in. 

Thank you for the PR. 

:money_with_wings:

File: mycli/main.py (hunk 1 above)
Comment (j-bennet) [inline mycli/main.py]: Only just saw the email. No, the values from files lower on the list should overrides previous values.

Comment (j-bennet) [inline mycli/main.py]: Only just saw the email. No, the values from files lower on the list should overrides previous values.

**Comment by derobert** [comment]
Thank you. Confirmed working here.

**Pull request merged**